1-2/main.js

```diff
@@ -0,0 +1,15 @@
+function quiz() {
+  let answer = true;
+  
+  while (answer) {
+    const reply = prompt('日本の首都は？');
+    if(reply !== '東京'){
```

Comment:
`()``{}`の両外側にはスペースをあけるようにしましょう。
他の部分も同様です。
```suggestion
    if (reply !== '東京') {
```